src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/VariantsToTable.java

```diff
@@ -28,60 +28,78 @@
+ * </ul>
  *
  * <h4>FORMAT/sample-level fields</h4>
- * <p>Use the `-GF` argument to extract FORMAT/sample-level fields, which will appear as a single column each per
- * sample in the output file.</p>
+ * <p>
+ *     Use the `-GF` argument to extract FORMAT/sample-level fields. The tool will create a new column per sample
+ *     with the name "SAMPLE_NAME.FORMAT_FIELD_NAME" e.g. NA12878.GQ, NA12877.GQ.
+ * </p>
  *
  * <h3>Inputs</h3>
  * <ul>
- *     <li>A VCF file from which to extract fields.</li>
- *     <li>A list of -F and/or -GF fields to extract.</li>
+ *     <li> A VCF file from which to extract fields</li>
+ *     <li> Fields to extract from a vcf. Each field must be listed separately with a -F and -GF flag </li>
  * </ul>
  *
  * <h3>Output</h3>
  * <p>
- * A tab-delimited file containing the values of the requested fields in the VCF file.
+ *     A tab-delimited file containing the values of the requested fields in the VCF file.
  * </p>
  *
  * <h3>Usage example</h3>
  * <pre>
- *     ./gatk-launch VariantsToTable \
+ *     gatk VariantsToTable \
  *     -V input.vcf \
- *     -F CHROM -F POS -F ID -F QUAL -F AC \
+ *     -F CHROM -F POS -F TYPE -GF AD \
  *     -O output.table
  * </pre>
  * <p>would produce a file that looks like:</p>
  * <pre>
- *     CHROM    POS ID      QUAL    AC
- *     1        10  .       50      1
- *     1        20  rs10    99      10
+ *     CHROM  POS        TYPE   HSCX1010N.AD  HSCX1010T.AD
+ *     1      31782997   SNP    77,0          53,4
+ *     1      40125052   SNP    97,0          92,7
+ *     1      65068538   SNP    49,0          35,4
+ *     1      111146235  SNP    69,1          77,4
  * </pre>
  *
- * <h3>Caveats</h3>
+ * <h3>Notes</h3>
  * <ul>
- *     <li>Some annotations cannot be applied to all variant sites, so VCFs typically contain records where some annotation values are missing. By default this tool will emit the special value NA for the missing annotations if you request export of an annotation for which not all records have values. You can override this behavior by setting --errorIfMissingData in the command line. As a result, the tool will throw an error if a record is missing a value.</li>
- *     <li>When you request export of FORMAT/sample-level annotations (such as GT), the annotations will be identified per-sample. If multiple samples are present in the VCF, the columns will be ordered alphabetically by sample name (SM tag).</li>
+ *     <li> It is common that certain annotations are absent for some variants. By default this tool will emit NA for an annotation that's missing. If you wish that the tool fail when it encounters a missing annotation, turn on the --error-if-missing-data flag in the command line. </li>
+ *     <li> If multiple samples are present in the VCF, the genotype fields will be ordered alphabetically by sample name (SM tag). </li>
+ *     <li> Filtered sites are ignored by default. To include them, turn on the --show-filtered flag. </li>
```

Comment:
--> The tool ignores filtered sites with annotations in the FILTER column and only considers those records with `.` or `PASS` in the FILTER column. To include filtered sites, specify the --show-filtered option.

Reply:
My version is a bit terse, but I think it's descriptive enough. Reworded a bit to make it clearer.